extern/storage-sealing/sealing.go

```diff
@@ -140,7 +141,16 @@ func (o *openSector) dealFitsInLifetime(dealEnd abi.ChainEpoch, expF func(sn abi
 	return expiration >= dealEnd, nil
 }
 
+type pieceAcceptResp struct {
+	sn     abi.SectorNumber
+	offset abi.UnpaddedPieceSize
+	err    error
+}
+
 type pendingPiece struct {
+	doneCh chan struct{}
+	resp   atomic.Value
```

Comment:
I don't think resp needs to be an atomic value - the doneCh prevents concurrent access

Reply:
good point. fixed.